packages/module/src/pipelines/components/groups/DefaultTaskGroup.tsx

```diff
@@ -117,13 +117,15 @@ const DefaultTaskGroupInner: React.FunctionComponent<DefaultTaskGroupInnerProps>
       return [0, 0];
     }
     switch (labelPosition) {
+      case LabelPosition.top:
+        return [minX + (maxX - minX) / 2, -minY + (padding + labelOffset) * 2];
```

Comment:
```suggestion
        return [minX + (maxX - minX) / 2, -minY + labelOffset)];
```

Reply:
Ah okay... I made some changes later on but missed to update this spot. I have updated it now. Thanks